travis/pg_travis_multi_test

```diff
@@ -12,16 +12,31 @@ fi
 status=0
 testtargets="$*"
 
-# Configure, build, and install extension
-./configure --enable-coverage PG_CONFIG=/usr/lib/postgresql/$PGVERSION/bin/pg_config
+# If USE_CUSTOM_PG_BUILD variable is not set, we will assume custom build is
+# not required. If custom build is not required, that means PostgreSQL was
+# installed via apt, otherwise we compiled it from the source. Depending
+# on the situation, location of pg_config is different.
+if [ -z "$USE_CUSTOM_PG_BUILD" ] || ! [ $USE_CUSTOM_PG_BUILD ]; then
+  ./configure --enable-coverage PG_CONFIG=/usr/lib/postgresql/$PGVERSION/bin/pg_config
+else
+  ./configure --enable-coverage PG_CONFIG=/usr/local/pgsql/bin/pg_config
+fi
```

Comment:
Seems like it'd be better to set a $PG_CONFIG or such in the above branches, rather than repeating it later.  That way we don't need to maintain two branches here.